intro-HPC/leuven/ch_hpc_policies.tex

```diff
@@ -0,0 +1,77 @@
+\chapter{HPC Policies}
+\label{ch:hpc-policies}
+
+The cluster has been setup to support the ongoing research in all the domains
+at the \university.  As such, it should be considered as
+valuable research equipment.  User shall not abuse the system for other purposes. By
```

Comment:
You are using the plural form in the next sentence. Maybe also use plural `Users` here?